fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/common/StoreTimer.java

```diff
@@ -167,6 +167,13 @@ protected static Counter getCounter(@Nonnull Map<Event, Counter> counters, @Nonn
          * @return the user-visible title
          */
         String title();
+
+        /**
+         * Get the logKey of this event for logging purposes.
```

Comment:
Maybe just "key" instead of "logKey"? And/or adding text that this should be used with `KeyValueLogMessage`s and other key-value based logging systems to log the values from instrumented events.

We may also want to add something about how often we expect these to change. Like, should the user rely on the log key not being updated, or do we feel like we can change this whenever we see fit?

Reply:
Done. Please review.